src/njoy21/input/DTFR/Card3a/Mtc.hpp

```diff
@@ -0,0 +1,14 @@
+struct Mtc {
+  using Value_t = int;
+  static std::string name() { return "mtc"; }
+  static std::string description() {
+    return
+      "Mtc is the MT number for thermal coherent cross sections to be\n"
+      "extracted from the GENDF tape. As with all MT numbers, Mtc must be\n"
+      "three digits or less.";
+  }
+  
+  static Value_t defaultValue(){ return 0; }
+  static bool verify( Value_t i ) { return i <= 999 && i >= 0; }
```

Comment:
In the manual, on page 393, it says that card3a "gives the mt numbers for the thermal
incoherent and coherent cross sections," where examples of this include "221 and zero for free-gas scattering," which presumably means mti = 221, mtc = 0. So I think that if you choose to not include either incoherent or coherent cross sections, you can set it equal to zero